index.html

```diff
@@ -53,7 +53,7 @@ <h3>Project name goes here</h3>
           <a href="#">
             <button type="button">
               <span>See this project</span>
-              <img class="pr-button-img" src="./assets/Union.png" />
+              <img class="pr-button-img" src="./assets/Union.png" alt="button" />
```

Comment:
- [OPTIONAL] The alternative text does not need to include the words "button", "link", or "image of". (Screen readers automatically provide that information.). Kindly find a better text for your `alt`. I intentionally decided to not request for changes, because other requirements are fine. I highly recommend you to attend to this and make it habit in your  futures projects to avoid this common mistake.